src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Processors/Settings/GeneralizeSetting.cs

```diff
@@ -23,6 +23,15 @@ public static GeneralizeSetting CreateFromRuleSettings(Dictionary<string, object
             try
             {
                 cases = JObject.Parse(ruleSettings.GetValueOrDefault(RuleKeys.Cases)?.ToString());
+                
+                FhirPathCompiler compiler = new FhirPathCompiler();
+
+                // Validate cases for compiling.
+                foreach (var eachCase in cases)
+                {
+                    compiler.Compile(eachCase.Key.ToString());
+                    compiler.Compile(eachCase.Value.ToString());
+                }
             }
             catch (Exception ex)
             {
```

Comment:
possible to get the detailed failed case?

Reply:
Fixed it.